### PR TITLE
Stop push if not up to date with latest

### PR DESCRIPTION
**Pull request by pjlee11**
_Takes the most recent commit from origin/latest and checks for it in a list of all commits in the history of the branch_

* [x] Fixes https://github.com/bbc/simorgh/issues/261
* ~[ ] Tests added for new features~

* ~[ ] Test engineer approval~

Example of failure:
```
git log origin/latest -n 1 --pretty=format:"%H" // (the most recent commit on origin/latest) 43591841a34a500a718ee03d33397912871ef2e7
git log --pretty=format:"%H" /* (all the commit history of this branch)
			0d4df4bbabdcddace51724d4a354ab98dc52f847
			4ea650b0a1dd4350d1081ed3fd4206043781c086
			6397453dee3cb35e11b79f6d5c5c6ee8c27ca98c
			34fa4ac856816b4d84e858228fad8d2c87e73073
			289eba6f4cdee1310b9591bad54e0c150779111b
                        ...
			*/
		
sh upToDateWithLatest.sh // (denies the push by exiting the current process)
Your branch is not up-to-date with latest. Push to origin denied.
Please pull in the most recent version of origin/latest
```

### CODE REVIEW

**Comment by pjlee11** [comment]
**Output example**

image

**Review by BogdanDogaru — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -11,9 +11,9 @@
     "cypress:storybook:interactive": "cypress open --project ./.storybook/.",
     "dev": "razzle start",
     "lighthouse": "lighthouse http://localhost:7080/article/id --output-path=./reports/report.html --view",
-    "precommit": "npm run test:lint",
-    "prepush": "npm run test:unit",
     "postmerge": "npm install",
+    "precommit": "npm run test:lint",
+    "prepush": "./upToDateWithLatest.sh && npm run test:unit",
```
Comment (jtart) [inline package.json]: Please move this to be inside a `scripts` dir.

**Review by jtart — CHANGES_REQUESTED** [review]
1 minor change. Interested to see it in action so would like to see it merged ASAP.

**Comment by sareh** [comment]
Agreed that the file should be in a `scripts` directory. When I tried this locally (after making the script file an executable file), I got this error, as expected.
```
Your branch is not up-to-date with latest - push to origin denied.
Please pull in the most recent version of origin/latest. EG: 'git pull origin latest'

husky > pre-push hook failed (add --no-verify to bypass)
error: failed to push some refs to 'git@github.com:bbc/simorgh.git'
```

File: package.json (hunk 1 above)
Comment (pjlee11) [reply package.json]: Done 😎

**Review by sareh — DISMISSED** [review]

**Review by sareh — APPROVED** [review]
Looks good!

**Review by jtart — APPROVED** [review]

**Pull request merged**